internal/corazawaf/telemetry.go

```
@@ -0,0 +1,142 @@
+	if sink == nil {
+		return
+	}
+	sink.OnEngineError(plugintypes.EngineEvent{
+		Kind:          kind,
+		TransactionID: tx.id,
+		Phase:         tx.lastPhase,
+		Err:           err,
+		Message:       msg,
+	})
+}
+
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)